prow/cmd/checkconfig/main.go

```diff
@@ -517,7 +517,7 @@ func strictBranchesConfig(c config.ProwConfig) (*orgRepoConfig, error) {
 }
 
 func validateStrictBranches(c config.ProwConfig) error {
-	const explanation = "See #5: https://github.com/kubernetes/test-infra/blob/master/prow/cmd/tide/maintainers.md#best-practices Also note that this validation is imperfect, see the check-config code for details"
+	const explanation = "See #5: https://docs.prow.k8s.io/docs/components/core/tide/maintainers/#best-practices Also note that this validation is imperfect, see the check-config code for details"
```

Comment:
This modification is causing `pull-prow-unit-test` to fail. ([error log](https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_prow/103/pull-prow-unit-test/))

I guess that the `checkconfig` is intended to accept GitHub blob or tree URLs of `kubernetes`/`k8s` org,
and this causes error because I am trying to replace with `docs.prow.k8s.io` URL, which is not a GitHub URL.

We might
1. Modify `checkconfig` code (or its test code) to accept `docs.prow.k8s.io` URLs as well
   (but I guess that this is against the original intention of `checkconfig`, so not appropriate)
2. Or, update URLs in L520 & L980 to https://github.com/kubernetes-sigs/prow/blob/main/site/content/en/docs/components/core/tide/maintainers.md#best-practices
   (We might need to modify `checkconfig` code to accept `kubernetes-sigs` GitHub org as well.)